test/mitmproxy/proxy/protocol/test_http2.py

```diff
@@ -508,8 +505,10 @@ def handle_server_event(cls, event, h2_conn, rfile, wfile):
         return True
 
     def test_body_size_limit(self):
-        self.config.options.body_size_limit = "20"
-        self.config.options._processed["body_size_limit"] = 20
+        self.options.body_size_limit = "20"
+
+        # FIXME: This should not be required?
+        self.options._processed["body_size_limit"] = 20
```

Comment:
This is from https://github.com/mitmproxy/mitmproxy/commit/201c65960eb19b97e6c499f22082aa176bdaa6b1#diff-ae3ce6b83d0e48d6158f2e5fe6583e6b
we aren't using the core addon which probably sets it